src/explore-education-statistics-common/src/services/types/blocks.ts

```diff
@@ -93,7 +93,7 @@ export interface DataBlock extends BaseBlock {
   heading: string;
   source: string;
   dataBlockRequest: DataBlockRequest;
-  charts: Chart[];
+  chart?: Chart;
```

Comment:
De-listing charts is actually a bad idea. A future feature is to be able to have multiple charts per data block. See: https://dfedigital.atlassian.net/browse/EES-753

I would revert all of the relevant changes.